CodeStarterCamp_Week4/Exercise.swift

```diff
@@ -9,56 +9,60 @@ import Foundation
 
 struct Exercise {
         let name: String
-        let action: () -> Void
+        let action: (BodyCondition) -> Void
 }
 
 // 상체운동
-let sitUp = Exercise(name: "윗몸일으키기") {
+let sitUp = Exercise(name: "윗몸일으키기") { (person: BodyCondition) in
     person.upperBodyStrength += Int.random(in: 10...20)
     person.fatigue += Int.random(in: 10...20)
 }
```

Comment:
솜이 action 프로퍼티의 타입을 `(BodyCondition) → Void` 로 이미 정의하셨죠?
지금부터 이어질 운동들은.. 전부 Exercise 타입의 인스턴스 들이잖아요?

따라서, person 이라는 임시변수는 `당연히` BodyCondition 타입일 거에요.
그러니까, person 뒤에서 콜론(:) 찍고, 다시 타입 명시를 해줄 필요가 없습니다. 😄 

이런 거 할때, 이 상수/변수가 무슨 타입이지? 🤔 궁금하시다면...
`옵션` 키를 누른 상태에서, 상수/변수를 좌클릭 해보세요...! 그러면 `타입`을 빠르게 알 수 있어요. 🔍 

```suggestion
let sitUp = Exercise(name: "윗몸일으키기") { person in
    person.upperBodyStrength += Int.random(in: 10...20)
    person.fatigue += Int.random(in: 10...20)
}
```

Reply:
다른 운동들도 전부 이 방식으로 고쳐보세요!

Reply:
<img width="482" alt="스크린샷 2022-05-14 오후 5 50 08" src="https://user-images.githubusercontent.com/94514250/168418620-d0ab0886-c14e-4f9c-aa3c-bb68367da32c.png">

진짜 신기하네요!ㅎㅎ 나중에 복잡한 코드 만들 때 도움이 될 거 같아요👍
이미 person이 BodyCondition타입이라 따로 선언할 필요는 없었군요... 수정했습니다!